src/libs/HeaderUtils.ts

```diff
@@ -15,7 +18,19 @@ function getPinMenuItem(report: OnyxReport): ThreeDotsMenuItem {
     };
 }
 
+function getShareMenuItem(report: OnyxReport, participants: number[]): ThreeDotsMenuItem {
+    const isGroupDMChat = ReportUtils.isDM(report) && participants.length > 1;
```

Comment:
Check for isGroupDMChat when we are deciding whether to add the share menu item or not. Then delete this method as it is only used in one place.